doc/nrf/ug_matter_configuring.rst

```diff
@@ -29,37 +24,34 @@ The following figure shows the protocol types and the available Matter controlle
 
    Protocol types and controllers used by Matter
 
-To enable the IPv6 communication with the Matter accessory device over the Thread network, the Matter controller requires the Thread Border Router.
-This is because the Matter controller types described on this page do not have an 802.15.4 Thread interface.
-The Border Router bridges the Thread network with the network interface of the controller, for example Wi-Fi.
+Matter development environment depends on the following criteria:
+
+* Network type:
 
-Matter development environment setup options depend on how you run the Matter controller.
-You can either run it on the same device as the Thread Border Router or run the Matter controller and the Thread Border Router on separate devices.
+  * To enable the IPv6 communication with the Matter accessory device over the Thread network, the Matter controller requires the Thread Border Router.
+    This is because the Matter controller types described on this page do not have an 802.15.4 Thread interface.
+    The Border Router bridges the Thread network with the network interface of the controller, for example Wi-Fi.
+    Watch the following video for an overview of the Matter over Thread setup.
 
-Depending on your choice, you can set up one of the following development environments:
+    .. raw:: html
 
-* Thread Border Router on Raspberry Pi and Matter controller on PC (CHIP Tool for Linux or macOS)
+       <div style="text-align: center;">
+       <iframe width="560" height="315" src="https://www.youtube.com/embed/XWWJYraNZPs" title="Matter development environment video overview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+       </div>
 
-  .. include:: /ug_matter_configuring_env.rst
-     :start-after: matter_env_ctrl_pc_start
-     :end-before: matter_env_ctrl_pc_end
 
-* Thread Border Router on Raspberry Pi and Matter controller for mobile (CHIP Tool for Android)
+  * To enable the IPv6 communication with the Matter accessory device over the Wi-Fi network, you need a Wi-Fi Access Point, for example a Wi-Fi router, so that the Matter controller can interact with any Wi-Fi device directly.
 
-  .. include:: /ug_matter_configuring_env.rst
-     :start-after: matter_env_ctrl_mobile_start
-     :end-before: matter_env_ctrl_mobile_end
+* Matter controller setup:
```

Comment:
The entire choice is for Matter over Thread only. In Wifi you don't have the presented choice (to run it on TBR or separate device). Using the word `separate` alone without telling `from what` seems confusing to me.